### DISCUSSION TITLE
My stuff on Resource Pages

### DISCUSSION THREAD

**Discussion by l-yk-l** [filamentphp/filament#3979] [Help] [answered]
Good afternoon!

I need to add my widgets on the resource pages between the form and relation managers.
Are there any easy ways to do this?
If not, is there any way I can change the blade resource to display these pages with my widgets?

Thank you!

**Comment by danharrin** [comment, marked as answer]
Maybe if you combined the form and relation managers into tabs, then the widget placement wouldnt be bad?

https://filamentphp.com/docs/2.x/admin/resources/relation-managers#moving-the-resource-form-to-tabs

**Reply by l-yk-l** [reply]
Thank you! I'll try it!

**Reply by l-yk-l** [reply]
I can't find this method in ViewRecord/EditRecord classes or in their ancestors

**Reply by danharrin** [reply]
Make sure you're on the latest Filament version

**Reply by l-yk-l** [reply]
Ah yep, my bad.
But how I can rename this pane?
![image](https://user-images.githubusercontent.com/33335112/189687021-6cd4dbea-8368-4893-83f9-8325ef11a006.png)

**Reply by danharrin** [reply]
Override the `getFormTabLabel()` method

**Reply by l-yk-l** [reply]
Thank you!
